python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -957,6 +987,32 @@ def _get_repo_job_by_name(context: SensorEvaluationContext, job_name: str) -> Jo
 
         return resolved_run_requests
 
+    def resolve_backfill_requests(
+        self,
+        run_requests: Sequence[RunRequest],
+        context: SensorEvaluationContext,
+    ) -> Sequence[RunRequest]:
+        for run_request in run_requests:
+            asset_selection = check.not_none(
+                self._asset_selection,
+                "Can only yield RunRequests with asset_graph_subset for sensors with an asset_selection",
+            )
+            check.invariant(
+                run_request.asset_graph_subset is None,
```

Comment:
shouldn't this be `is not None`?

Reply:
yep. i get `invariant` wrong every time, it's fixed now